### ISSUE TITLE
Update to support Rollup v4

### ISSUE THREAD

**Issue by mririgoyen** [Ephem/rollup-plugin-preserve-directives#17]
After updating to Rollup v4, we are starting to get peer dependency warnings.

```
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: redacted
npm WARN Found: rollup@3.29.4
npm WARN node_modules/rollup
npm WARN   dev rollup@"~4.0.2" from the root project
npm WARN   9 more (@rollup/plugin-commonjs, @rollup/plugin-terser, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer rollup@"2.x || 3.x" from rollup-plugin-preserve-directives@0.2.0
npm WARN node_modules/rollup-plugin-preserve-directives
npm WARN   dev rollup-plugin-preserve-directives@"~0.2.0" from the root project
```

**Comment by vrajpal-jhala** [comment]
To suppress warnings for now.

`package.json`
```
"overrides": {
    "rollup-plugin-preserve-directives": {
      "rollup": "4.6.0"
    }
},
```

**Comment by zrg** [comment]
Looks like there is a fork to resolve this. Any collaborators able to open a PR?
https://github.com/Ephem/rollup-plugin-preserve-directives/compare/main...sashathor:rollup-plugin-preserve-directives:main

**Comment by Ephem** [comment]
Sorry for the silence here. Thanks to @lachlancollins contribution in #19 this has been fixed and is now out in 0.3.0! 🎉
